### DISCUSSION TITLE
Welcome to UI Discussions!

### DISCUSSION THREAD

**Discussion by CM000n** [UI-Lovelace-Minimalist/UI#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KaherdinTristan** [comment]
Ok, I will start... 
- A small, "or at least not so big" buttons, that can be used as wanted. Ie. Small button to change the colors of a led list to a specific color 
- A pop-up template to add info or toggles to, that is in line with the Minimalistic UI? 
=) 🕊️ ✌️

**Comment by KaherdinTristan** [comment]
_I have had this idea for a while, but I have not found time to summarize it before. And I don't know any coders... so this forum is the closest I have gotten to getting to know You guys that actually know coding... so here I go_

- To enable the user to create a card that fills their needs, or just to enable the user to modify an existing card
### Card Creator
An editor that displays toggles, icons, sliders, value fields that initially are static/mockups, and after being dragged onto an empty card can be connected to an entity or device. Furthermore the card itself can be modified regarding color, states, borders, etc. Conditions can be set for the card or the individual toggles/fields. This new card can be saved as draft, or for use. The card can be edited or deleted.
A preview is being shown with real values from Your instance of HA if you want to. 

![image](https://user-images.githubusercontent.com/3398594/146529863-0381572c-e6db-49b8-8cf9-40d5e121e73f.png)
Image 1. "_Hand drawn shitty sketch_"

**Question**
What would it take to make such a tool?
😄 🚀 🌔

**Reply by CM000n** [reply]
A very cool idea! Do you mean a card designer for a specific theme with that, or a card designer to really freely create any cards? I think the second will be much more difficult with the current interaction of themes and templates etc. in Home Assistant.

**Reply by KaherdinTristan** [reply]
My _initial idea_ was based on "Lovelace built in cards", in other words _without_ any themes applied.
In a **Dream Scenario** the "program" would be built with themes in mind, so that someone in the future could add themes to it. 

But, could this be done you think? And on what kind of plattform?

**Reply by CM000n** [reply]
In IT, almost anything can be done. The only question is how much effort it will cost ;-)
Unfortunately, my programming skills are not sufficient for this at the moment. More than a little templating and python knowledge I don't have.
Maybe this would also be a general topic for Homeassistant and the frontend developers there?

**Reply by KaherdinTristan** [reply]
Ok, so I've posted on [HA community forum\dev](https://community.home-assistant.io/t/frontend-devs-do-you-want-a-challenge/368352) now. 
I hope they don't make too much fun of me there...

**Reply by CM000n** [reply]
Good Lück :-) I will also keep an eye in the thread.

**Comment by KaherdinTristan** [comment]
Is the door/window template implemented in the template code Simon? @CM000n 
I was thinking of adding it otherwise... 
https://community.home-assistant.io/t/lovelace-ui-minimalist/322687/50? 

Edit: my bad. My sensors had not been configured as "door"
Devices\
![2](https://user-images.githubusercontent.com/3398594/146645475-680f4ab7-58b1-4017-ac2c-ebe0259008f4.png)

**Reply by CM000n** [reply]
There are currently only the "`card_cover_with_buttons`" for controlling roller shutters and the general cards for binary sensors. But there are no dedicated cards for binary window or door sensors yet. So that would certainly be a cool extension.

**Reply by KaherdinTristan** [reply]
![Screenshot_20211219_085407.jpg](https://user-images.githubusercontent.com/3398594/146667817-36271228-b947-4c68-aa5b-fa32e8eb472f.jpg)

This is what i got it the sensor was categorized as door, and smoke sensors... And motion sensors

**Reply by CM000n** [reply]
Sorry, but I don't understand what you're trying to tell me with this. :-D This is just the display for our general cards for binary sensors in your Screenshot.

The one you linked at the beginning of the thread is a special card from clemalex, for doors and windows, with its own icon and color scheme.

**Reply by KaherdinTristan** [reply]
> Sorry, but I don't understand what you're trying to tell me with this. :-D This is just the display for our general cards for binary sensors in your Screenshot.
> 
> The one you linked at the beginning of the thread is a special card from clemalex, for doors and windows, with its own icon and color scheme.

Yeah, 😆 I noticed that is was just that after trying them out for a while... 
So, yes, I was thinking of a card with just that; colored doors when opened etc. 

Btw; where do You think I should start if I want to learn about building and designing cards of my own? 😀

**Reply by CM000n** [reply]
I think the best way in the beginning is simply via trial an error. Just fork this, or other repos, try to understand things and just build some of your own cards to get an understanding of how things work together.

**Comment by magicbenny** [comment]
HI all,

i rly like this minimalistic style. but just in case of the chips I am asking me for a while how to change the state icon for a chip. just in case the binary sensor is a motion-sensor I would like to use a different icon than the "bed" icon :)

how can this be changed?

**Reply by magicbenny** [reply]
is it rly that simple to use the emoji keyboard? 🤣
